MathPix/TeXEditorWindow.xaml

```diff
@@ -65,8 +66,14 @@
             <Button Name="BtnInsertTags"
                     Grid.Column="1"
                     VerticalAlignment="Center"
-                    Content="Insert Tags"
-                    Click="BtnInsertTags_Click" />
+                    Click="BtnInsertTags_Click">
+              <Button.Content>
+                <TextBlock>
+                  INSERT <Underline>T</Underline>AGS
```

Comment:
That's a valid approach. FYI, there is an integrated way of doing this without code behind: https://github.com/supermemo/SuperMemoAssistant/blob/80ef5230d6ff7c25aa0668b60bfd8d960f064aba/src/AppHosts/SuperMemoAssistant/CollectionSelectionWindow.xaml#L105

Reply:
Ahh got it, I'll switch to the <AccessText> method, it's simpler.  The new version works, the old version would erase the selected text.